catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -47,178 +55,104 @@
-        if (validators == null || validators.size() < 1) {
-            console.println(ERROR_COLOR + "No validators have been configured" + COLOR_DEFAULT);
-            return false;
-        }
-        return true;
+        return bad;
     }
 
-    private Metacard createMetacard() {
-
-        // Although we are given XML, we need to put it in a metacard to work with the validators.
-        MetacardImpl metacard = null;
-        try {
-            String metadata = IOUtils.toString(new File(filename).toURI());
-            metacard = new MetacardImpl();
-            metacard.setMetadata(metadata);
-        } catch (IOException e) {
-            console.println(ERROR_COLOR + "Error reading file. Check the log for the stacktrace"
-                    + COLOR_DEFAULT);
-            LOGGER.error("Error trying to read file {}", filename, e);
+    private List<Metacard> createMetacards() throws IOException {
+        Collection<File> files = getFiles();
+        List<Metacard> metacards = new ArrayList<>();
+        for (File file : files) {
+            Metacard metacard = new MetacardImpl();
+            String metadata = IOUtils.toString(file.toURI());
+            metacard.setAttribute(new AttributeImpl(Metacard.METADATA, metadata));
+            metacard.setAttribute(new AttributeImpl(Metacard.TITLE, file.getName()));
+            metacards.add(metacard);
         }
-        return metacard;
+        return metacards;
     }
 
-    private void runValidators(Metacard metacard) {
-
-        // In an effort to make the output of this command more readable on the console,
-        // we model it after the 'tree' command in Linux. We are printing a hierarchy of lists,
-        // so we want to make the last element of the list pronounced so that we can more
-        // easily see the groupings of the hierarchy. This is why we use an iterator
-        // and if-else rather than a more simple for-loop.
-        Iterator<MetacardValidator> iterator = validators.iterator();
-        while (iterator.hasNext()) {
-            MetacardValidator validator = iterator.next();
-            if (iterator.hasNext()) {
-                printValidator(validator, ELEMENT);
-                printErrorsAndWarnings(validator, metacard, INDENT);
-            } else {
-                printValidator(validator, LAST_ELEMENT);
-                printErrorsAndWarnings(validator, metacard, LAST_INDENT);
-            }
-        }
-    }
+    private List<Metacard> getMetacardsFromCatalog() throws Exception {
+        List<Metacard> results = new ArrayList<>();
 
-    private void printValidator(MetacardValidator validator, String prefix) {
-        String name = validator.getClass()
-                .getName();
-        if (validator instanceof Describable && ((Describable) validator).getId() != null) {
-            name = ((Describable) validator).getId();
-        }
-        prettyPrint(prefix, name);
-    }
+        Filter filter = builder.attribute(Metacard.ANY_TEXT)
```

Comment:
Should probably accept CQL strings instead. I think that's what the catalog commands do.